### PR TITLE
Week5 submission

### PR DESCRIPTION
**Pull request by duokobia**
This is my submission for week 5.

### CODE REVIEW

File: 03-task-manager/starter/app.js [hunk 1]
```diff
@@ -1 +1,39 @@
...
+const start = async() => {
+    try {
+        await connectDB(process.env.MONGO_URI)
+        app.listen(port, console.log(`Server is listening on port ${port}...`))
+
+    } catch (error) {
+        console.log(error)
+    }
+};
+// to access the dotenv anywhere, use process.env.<packageName>
```
Comment (taylor-knapp) [inline 03-task-manager/starter/app.js:37]: Just to clarify - `dotenv` is a library that adds the variables in the provided `.env` file to the built-in Node `process.env` objection. When you are calling `process.env.VARIABLE` you are just accessing an object with properties on it. (dotenv is only used to modify that object).

File: 03-task-manager/starter/models/Task.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const mongoose = require("mongoose");
+
+// Only the properties set in your Schema will be passed unto the database.
+// Everything else will be ignored. When we use schema there will be a structure in our DB.
+// Model is a representation for the collection. So all the tasks to be pushed into our DB
+// will be added into that collection(model).
+const TaskSchema = new mongoose.Schema({
+    name: String,
+    completed:Boolean
+});
```
Comment (taylor-knapp) [inline 03-task-manager/starter/models/Task.js:10]: This works -- and, you can define each property in the schema as an object that includes additional validation and defaults. [Take a look at the example to see how you can extend this.](https://github.com/duokobia/node-express-course/blob/main/03-task-manager/final/models/Task.js)

**Review by taylor-knapp — APPROVED** [review]
Great work! 🎉 🥳

File: 03-task-manager/starter/models/Task.js (hunk 2 above)
Comment (duokobia) [reply 03-task-manager/starter/models/Task.js:10]: Great.

Thank you for the link as well.

Comment (duokobia) [reply 03-task-manager/starter/models/Task.js:10]: Great.

Thank you for the link as well. I saw how John used it.

**Pull request merged**